tests/pytests/test_vecsim.py

```diff
@@ -227,53 +227,74 @@ def testDelReuseLarge(env):
 def testCreate(env):
     env.skipOnCluster()
     conn = getConnectionByEnv(env)
-    conn.execute_command('FT.CREATE', 'idx1', 'SCHEMA', 'v', 'VECTOR', 'FLOAT32', '1024', 'IP', 'HNSW', 'INITIAL_CAP', '10', 'M', '16', 'EF', '200')
-    info = [['identifier', 'v', 'attribute', 'v', 'type', 'VECTOR', 'TYPE', 'FLOAT32', 'SIZE', '1024', 'METRIC', 'IP', 'ALGORITHM', 'HNSW', 'M', '16', 'EF CONSTRUCTION', '200']]
+    conn.execute_command('FT.CREATE', 'idx1', 'SCHEMA', 'v', 'VECTOR', 'HNSW', '14', 'TYPE', 'FLOAT32', 'DIM', '1024', 'DISTANCE_METRIC', 'IP', 'INITIAL_CAP', '10', 'M', '16', 'EF_CONSTRUCTION', '200', 'EF_RUNTIME', '10')
+    info = [['identifier', 'v', 'attribute', 'v', 'type', 'VECTOR', 'ALGORITHM', 'HNSW', 'TYPE', 'FLOAT32', 'DIM', '1024', 'DISTANCE_METRIC', 'IP', 'M', '16', 'EF_CONSTRUCTION', '200', 'EF_RUNTIME', '10']]
     assertInfoField(env, 'idx1', 'attributes', info)
 
-    conn.execute_command('FT.CREATE', 'idx2', 'SCHEMA', 'v', 'VECTOR', 'FLOAT64', '4096', 'L2', 'HNSW', 'INITIAL_CAP', '10', 'M', '32', 'EF', '100')
-    info = [['identifier', 'v', 'attribute', 'v', 'type', 'VECTOR', 'TYPE', 'FLOAT64', 'SIZE', '4096', 'METRIC', 'L2', 'ALGORITHM', 'HNSW', 'M', '32', 'EF CONSTRUCTION', '100']]
-    assertInfoField(env, 'idx2', 'attributes', info)
+    # Uncomment these test when support for FLOAT64, INT32, INT64, is added.
```

Comment:
Those tests should cause failure for now. What will be the outcome if someone will try to run those commands?

Reply:
We're testing for bad type valuse in line 279.
Right now these values will also result in the same error, but these tests will pass when we'll support them.
I can add tests for that in the 'invalid init args' section and comment to remove them when we add the right support.